components/create-collective/sections/CreateCollectiveForm.js

```diff
@@ -149,7 +164,7 @@ class CreateCollectiveForm extends React.Component {
 
     return (
       <Flex flexDirection="column" m={[3, 0]}>
-        {host && <CreateCollectiveCover host={host} />}
+        {host && query.hostCollectiveSlug !== 'opensource' && <CreateCollectiveCover host={host} />}
```

Comment:
That's not the ideal solution.

Reply:
We show the Collective navbar on the form for Collectives with a host, unless the host is the Open Source Collective (becaues they are coming from manual verification). So this makes sense to me in a straightforward way. What would you suggest as an alternative?

Reply:
Have discussed via Slack